addons/web/static/src/core/errors/error_handlers.js

```diff
@@ -135,9 +135,9 @@ function lostConnectionHandler(env, error, originalError) {
             env.services
                 .rpc("/web/webclient/version_info", {})
                 .then(function () {
-                    env.services.notification.close(connectionLostNotifId);
-                    connectionLostNotifId = null;
-                    env.services.notification.create(
+                    connectionLostNotifRemove();
```

Comment:
this looks unsafe. what if connectionLostNotifRemove is null at this point? wouldn't it be better to wrap it in a if:
```js
if (connectionLostNotifRemove) {
  connectionLostNotifRemove();
  connectionLostNotifRemove = null;
}
```